internal/install/command.go

```diff
@@ -28,6 +30,11 @@ var Command = &cobra.Command{
 	Short:  "Install New Relic.",
 	Hidden: true,
 	Run: func(cmd *cobra.Command, args []string) {
+		go func() {
+			<-utils.SignalCtx.Done()
+			os.Exit(1)
```

Comment:
Maybe we want `os.Exit(130)` for "standards" adherence?  https://tldp.org/LDP/abs/html/exitcodes.html